Plugins/SQL/Targets/SQLite.cpp

```diff
@@ -32,7 +32,7 @@ void SQLite::Connect(NWNXLib::ViewPtr<NWNXLib::Services::ConfigProxy> config)
 { 
     if (auto database = config->Get<std::string>("DATABASE"))
     {
-        m_dbName = (*database).c_str();
+        m_dbName = database->c_str();;
```

Comment:
double `;` :P. Doesn't hurt any, though.

Reply:
Ugh, it hurts me tho